src/data-parser.js

```diff
@@ -55,44 +55,44 @@ const getSimDataFromMessage = message => {
     );
 };
 
-const serializeSimData = simData =>
+const serializeSimData = ({simData, hlChar}) =>
     simData.msisdn
         ? ':' +
           (simData.flag || 'flag-aq') +
-          ': *' +
+          ': ' +
+          hlChar +
           simData.msisdn +
           ' ' +
           simData.provider +
           ' ' +
           simData.paymentModel +
-          '*'
+          hlChar
         : '';
 
 const serializeMessageId = message =>
     (message.msisdn && message.msisdn != '?') ||
     (message.channel != undefined && 'Channel ' + message.channel) ||
     '';
 
-const identifySim = ({message, simData}) =>
-    serializeSimData(simData) ? serializeSimData(simData) : serializeMessageId(message);
+const identifySim = ({message, simData, hlChar}) =>
+    serializeSimData({simData, hlChar}) ? serializeSimData({simData, hlChar}) : serializeMessageId(message);
```

Comment:
Should hlChar come from outside but be part of serialization here?

Reply:
I don't follow you, what do you mean?